requirements.txt

```diff
@@ -0,0 +1,18 @@
+-e git+https://github.com/ckan/ckan.git@638262b96feef48e56da0d95f8dd3686092c5ae3#egg=ckan
+ckanext-datarequests==0.3.2
+Flask==0.11.1
+Pylons==0.9.7
+pysolr==3.6.0
+python-dateutil==1.5
+pyutilib.component.core==4.6.4
+python-magic==0.4.12
+redis==2.10.5
+requests==2.11.1
+Routes==1.13
+rq==0.6.0
+selenium==3.6.0
+simplejson==3.10.0
+slackclient==1.0.9
+SQLAlchemy==0.9.6
+sqlalchemy-migrate==0.10.0
+sqlparse==0.2.2
```

Comment:
Hi @WNjihia. The `requirements.txt` file here will cause conflicts with other people's CKAN set up. If only for test purposes, we should name it something different like `requirements-test.txt`